portal-web/test/functional/com/liferay/portalweb/macros/Form.macro

```diff
@@ -856,7 +856,7 @@ definition {
 
 		AssertTextEquals(
 			locator1 = "Form#FIELD_REFERENCE",
-			value1 = "${key_fieldReference}");
+			value1 = "${fieldReference}");
```

Comment:
@MoryAlvesBR, please see my [previous comment](https://github.com/liferay-forms/liferay-portal/pull/700#discussion_r661800317) carefully. You should do the same as it's being done below:
```
key_fieldName = "${fieldName}";
```